packages/web/src/app/admin/products/ProductForm.component.tsx

```diff
@@ -0,0 +1,337 @@
+                    <input
+                        type="text"
+                        name="name"
+                        id="name"
+                        value={formData.name}
+                        onChange={handleChange}
+                        required
+                        className="mt-2 block w-full rounded-md border-0 px-3 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                    />
+                </div>
+
+                {/* Image Upload Section */}
+                <div>
+                    <label className="block text-sm font-medium leading-6 text-gray-900">Produktfoto</label>
+                    <div className="mt-2">
+                        {/* Conditional rendering: Show preview if an image exists. */}
+                        {imagePreview ? (
+                            <div>
+                                <div className="w-full max-w-xs overflow-hidden rounded-lg">
+                                    <Image
+                                        src={imagePreview}
+                                        width={400}
+                                        height={400}
+                                        alt="Produktvorschau"
+                                        className="h-auto w-full object-cover"
+                                    />
+                                </div>
+                                <button
+                                    type="button"
+                                    onClick={handleRemoveImage}
+                                    className="mt-2 rounded-md bg-white px-3 py-2 text-sm font-semibold text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50">
+                                    Bild ändern
+                                </button>
+                            </div>
+                        ) : (
+                            // Otherwise, show the file upload dropzone.
+                            <div
+                                className={`flex justify-center rounded-lg border border-dashed px-6 py-10 transition-all duration-200 ${
+                                    isDragging ? "border-2 border-indigo-600 bg-indigo-50" : "border-gray-900/25"
+                                }`}
+                                onDragEnter={(e) => {
+                                    e.preventDefault();
+                                    setIsDragging(true);
+                                }}
+                                onDragOver={(e) => e.preventDefault()}
+                                onDragLeave={(e) => {
+                                    e.preventDefault();
+                                    setIsDragging(false);
+                                }}
+                                onDrop={handleDrop}>
+                                <div className="text-center">
```

Comment:
This div must use the style attribute `pointer-events: none` because otherwise the blue border will disappear when hovering over the text while dragging a file.